### PR TITLE
Bats analogy

### PR DESCRIPTION
**Pull request by jlowryduda**

### CODE REVIEW

File: conceptnet5/vectors/evaluation/analogy.py [hunk 1]
```diff
@@ -305,7 +331,7 @@ def eval_google_analogies(vectors, subset='semantic', vocab_size=200000, verbose
         result = best_analogy_3cosmul(
             vectors, tframe, *prompt
         )
-        if result == answer:
+        if result == answer or result in answer:
```
Comment (rspeer) [inline conceptnet5/vectors/evaluation/analogy.py]: This seems extremely forgiving -- isn't this saying that an answer is correct if it's any substring of the correct answer?

File: conceptnet5/vectors/evaluation/analogy.py [hunk 2]
```diff
@@ -136,7 +137,42 @@ def read_turk_ranks_semeval2012(subset, subclass):
...
+            if '/' in right:
+                right = [i.strip() for i in right.split('/')]
+            else:
+                right = [i.strip() for i in right.split(',')]
+            pairs.append([left, right])
+
+    quads = []
+    for i in range(len(pairs)):
+        a_pair = pairs[i]
+        a_pair[1] = a_pair[1][0]  # select only the first element if b1 is a list of synonyms
```
Comment (rspeer) [inline conceptnet5/vectors/evaluation/analogy.py]: What if b1 isn't a list of synonyms? Is this replacing it by its first character?

Comment (jlowryduda) [reply conceptnet5/vectors/evaluation/analogy.py]: This was an incorrect comment. b1 is always a list, therefore it's safe to do `a_pair[1][0]`

File: conceptnet5/vectors/evaluation/analogy.py (hunk 1 above)
Comment (jlowryduda) [reply conceptnet5/vectors/evaluation/analogy.py]: You're right.

**Pull request merged**